crates/router/src/connector/bluesnap/transformers.rs

```diff
@@ -0,0 +1,304 @@
+use serde::{Deserialize, Serialize};
+use crate::{
+    core::errors,
+    pii::{self, PeekInterface, Secret},
+    types::{self, api, storage::enums},
+};
+
+//TODO: Fill the struct with respective fields
+#[derive(Default, Debug, Serialize, PartialEq)]
+#[serde(rename_all = "camelCase")]
+pub struct BluesnapPaymentsRequest {
+    amount: String,
+    credit_card: Card,
+    currency: String,
```

Comment:
```suggestion
    currency: storage_enums::Currency,
```
Currency needs to be of this type, and should be handled accordingly while building request body